mlflow/projects/__init__.py

```diff
@@ -133,6 +133,7 @@ def _run(
             parameters=parameters,
             experiment_id=experiment_id,
             cluster_spec=backend_config,
+            env_manager=env_manager,
         )
 
     elif backend_name == "kubernetes":
```

Comment:
Can we support the kubernetes backend as well?

Reply:
Sure!

Reply:
Is there anything we need to update for the kubernetes backend? What it does is just build an image and run a command using the image. `--no-conda` or `--env-manaeger` is just ignored.

Reply:
@harupy Good point - we don't need to do anything for Kubernetes backend / Docker Projects.

*However*, we *do* need to update Docker Image build logic for MLflow Models to support building a Docker image with virtualenv, instead of conda. I think we missed this during planning. Can we add it this week?

Specifically, `mlflow models --build-docker` always uses conda, and `mlflow sagemaker --build-and-push-container` always uses conda too. 

Relevant code points:

- https://github.com/mlflow/mlflow/blob/3fa4481852b4a06db3cbe841dde7d5365a155e1e/mlflow/models/docker_utils.py#L86 / https://github.com/mlflow/mlflow/blob/3fa4481852b4a06db3cbe841dde7d5365a155e1e/mlflow/models/docker_utils.py#L31-L38 

- https://github.com/mlflow/mlflow/blob/3fa4481852b4a06db3cbe841dde7d5365a155e1e/mlflow/pyfunc/backend.py#L268-L273 

I'm happy to help out with this if you don't have time! :)